ui/app/src/views/SinglePool.vue

```diff
@@ -57,6 +78,27 @@ export default defineComponent({
       return t.imageUrl;
     });
 
+    watch([address, fromSymbol], async () => {
```

Comment:
The data does not load when navigate from other pages. However it does when I reload the single pool page. I suggest wrapping this functionality then adding it to `onMounted` lifecycle method. See: RewardsPage.vue